index.html

```diff
@@ -6,7 +6,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="index.css" />
 
-    <title>Portifolio mobile</title>
+    <title>Portfolio</title>
```

Comment:
### OPTIONAL
- [ ] Kindly update page title in `head` to have a better  page title, for example: Mirelapitt's portfolio or Mirelapitt full-stack webdeveloper*